api/v1/helper.go

```diff
@@ -103,6 +103,7 @@ func IsSupportedModel(vendorId, deviceId string) bool {
 			return true
 		}
 	}
+	log.Info("IsSupportedModel():", "Unsupported model:", "vendorId:", vendorId, "deviceId:", deviceId)
```

Comment:
IMO, it's better to use log.Warning here and below to get more attention into these log messages

Reply:
I agree but if you look at this file, all "errors" are reported via `log.Info`. The imported logger can do Info or Error. I could create another logger with glog and call `glog.Warning`, but... Up to you at this point.

Reply:
@e0ne any thoughts?